src/app/main/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.html

```diff
@@ -96,12 +96,13 @@ <h2 class="personal-info-pop-up-title">{{ 'personal-info.pop-up-title' | transla
       <app-ubs-input-error [formElement]="addressComment" [controlName]="'addressComment'"> </app-ubs-input-error>
     </div>
   </div>
+
+  <div class="d-flex justify-content-center buttons" mat-dialog-actions>
+    <button class="secondary-global-button btn m-0 mr-2" mat-button (click)="onNoClick()" type="button">
+      {{ 'personal-info.pop-up-cancel' | translate }}
+    </button>
+    <button class="primary-global-button btn m-0" type="submit" (click)="addAdress()" [disabled]="!addAddressForm.valid || isDisabled">
```

Comment:
As I remember FormGroup instance should have an `invalid` property, so you can replace `!addAddressForm.valid` with `addAddressForm.invalid`

Reply:
But I'm not sure, maybe only FormControls have that property, so check it